src/components/Sections/SectionNovelty/index.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <section class="novelty">
+        <div class="novelty__box container">
+            <img src="../../../assets/new-coffee.png" alt="new-coffee picture"
+            class="novelty__picture">
+            <div class="novelty__describe">
+                <p class="novelty__before-title">What’s new?</p>
+                <h2 class="novelty__title">Meet our new coffee!</h2>
```

Comment:
You can use a h2 tag when you already have a h1 tag on the page, but since you don't, and given that you have several such titles on the page, I would suggest using the strong tag instead